lib/active_record/events/method_factory.rb

```diff
@@ -34,9 +34,13 @@ def class_methods
       attr_reader :options
       attr_reader :naming
 
-      def define_predicate_method(module_, naming)
+      def define_predicate_method(module_, naming, strategy = :presence)
```

Comment:
These methods are not called outside of this class, so we can remove default value from the arguments list.